### PR TITLE
[ADD] project_task_type_remaining_time_zero

### PR DESCRIPTION
**Pull request by victor-ditay**
- Forgoten feature on refactoring elico_project

### CODE REVIEW

**Comment by victor-ditay** [comment]
@elicoidal @seb-elico Please have a check

File: project_task_type_remaining_time_zero/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+=====================================
+
+This module extends the functionality of project task type to configure
+stages that should set reamining time to zero.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/Elico-Corp/{project_repo}/issues>`_. In case of trouble, please
```
Comment (elicoidal) [inline project_task_type_remaining_time_zero/README.rst]: update

File: project_task_type_remaining_time_zero/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
```
Comment (elicoidal) [inline project_task_type_remaining_time_zero/__init__.py]: remove copyright in `__init__.py`

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by victor-ditay** [comment]
@elicoidal

**Comment by elicoidal** [comment]
Can you check travis?

**Comment by victor-ditay** [comment]
> Can you check travis?

```
109.77s$ travis_run_tests

======== Testing test_flake8 ========

project_completion_report/__openerp__.py:25:1: W391 blank line at end of file

project_completion_report/report/project_completion_report.py:45:9: E128 continuation line under-indented for visual indent

project_completion_report/report/project_completion_report.py:163:1: W391 blank line at end of file

======== Testing test_pylint ========

************* Module auth_cas.res_config

auth_cas/res_config.py:10: [W0403(relative-import), ] Relative import 'pycas', should be 'auth_cas.pycas'

From https://github.com/Elico-Corp/odoo-addons

 * [new branch]      8.0        -> origin/8.0
```

@elicoidal I fixed the errors regards this module, but basic all the other modules already on the master branch would fail.
Let fix on an specific PR as actually I can't know how many of them would fail

**Comment by elicoidal** [comment]
I thought you already fixed the issue of pycas

**Comment by victor-ditay** [comment]
> I thought you already fixed the issue of pycas

@elicoidal I couldn't find time. Think now would finish it.

**Comment by victor-ditay** [comment]
> I thought you already fixed the issue of pycas

@elicoidal One thing is that I haven't spotted yet:

This would implies migrate to new API all modules pointed.
So is more than just small adjustment on Travis. Not a problem for me as far you know.

cc @seb-elico

**Comment by elicoidal** [comment]
@victormartinelicocorp IMO no need for new API in v8 (not mandatory by Travis).
Fixing the pycas import should be good enough

**Comment by victor-ditay** [comment]
> @victormartinelicocorp IMO no need for new API in v8 (not mandatory by Travis).
Fixing the pycas import should be good enough

@elicoidal Weird that was pointed by Travis, I really didn't want to modify those modules but start to raise errors for the non CamelCase class, use of cr.commit(), old api osv deprecated and other like files should not have execution rights.

**Comment by seb-elico** [comment]
@victormartinelicocorp This module has nothing to do with task **type**. It should be called `project_task_stage_remaining_time_zero`. Please refactor all references to `type`

**Review by seb-elico — CHANGES_REQUESTED** [review]
Refactor `type` to `stage`

**Comment by seb-elico** [comment]
@victormartinelicocorp FYI Travis is failing for all the repo, not just your PR. I think it's better to create a separate PR to fix Travis, later this one will become green automatically (might require a re-build though)

**Comment by victor-ditay** [comment]
> Refactor type to stage

@seb-elico Odoo but internally is called `type` but agree the name is not so good so I will change it.

**Comment by victor-ditay** [comment]
> @victormartinelicocorp FYI Travis is failing for all the repo, not just your PR. I think it's better to create a separate PR to fix Travis, later this one will become green automatically (might require a re-build though)

@seb-elico Is what I did (#196) then I refactored this PR. Is still failing on which PR?

**Comment by seb-elico** [comment]
> @seb-elico Odoo but internally is called type but agree the name is not so good so I will change it.

@victormartinelicocorp Well spotted! Indeed functional name is "Task Stage" and technical name is `project.task.type`.

@elicoidal Which name seems the best? Maybe call the module "Project Task Stage - Set remaining time to zero" and keep the technical name `project_task_type_remaining_time_zero`

> @seb-elico Is what I did (#196) then I refactored this PR. Is still failing on which PR?

Maybe you need to re-build this one now (simply add 1 commit). But actually Travis is already green at the bottom!

**Comment by elicoidal** [comment]
> Which name seems the best? Maybe call the module "Project Task Stage - Set remaining time to zero" and keep the technical name project_task_type_remaining_time_zero

What about: project_task_set_remaining_time_zero

**Comment by elicoidal** [comment]
What are we waiting for?

**Comment by victor-ditay** [comment]
> What are we waiting for?

@seb-elico Could you check to approve it?

**Pull request merged**